src/main/java/menagerie/gui/Main.java

```diff
@@ -230,19 +232,19 @@ static void loadVLCJ(String path) {
     private List<Image> getIcons() {
         List<Image> results = new ArrayList<>();
         try {
-            results.add(new Image(getClass().getResourceAsStream("/icons/128.png")));
+            results.add(new Image(Objects.requireNonNull(getClass().getResourceAsStream("/icons/128.png"))));
```

Comment:
[matter of taste] personally, I like it more to have the nonnull-checks at the beginning of the methods

applies to all methods in this PR

Reply:
Agree, but since it's only for resource loading we might want to introduce a new method to avoid code duplication anyway.

So I'd say let's just leave it like this for now.